instrumentation/metric/histogram.go

```diff
@@ -81,15 +81,19 @@ func (h *Histogram) Rotate() {
 	h.histo.Rotate()
 }
 
-func (h histogramExport) LogRow() []*log.Field {
+func (h *histogramExport) LogRow() []*log.Field {
+	if h.Samples == 0 {
+		return nil
+	}
+
 	return []*log.Field{
 		log.String("metric", h.Name),
 		log.String("metric-type", "histogram"),
-		log.Int64("min", h.Min),
-		log.Int64("p50", h.P50),
-		log.Int64("p95", h.P95),
-		log.Int64("p99", h.P99),
-		log.Int64("max", h.Max),
+		log.Float64("min", h.Min),
```

Comment:
convert to float millis here; you can create a function `toMillisFloat` that takes an in64 and handles both casting and division by 1000000

Reply:
It can't take int64 because it also has to take h.Mean() which returns float.